crates/walrus-service/src/client/cli/args.rs

```diff
@@ -1055,12 +1055,13 @@ pub struct CommonStoreOptions {
     #[arg(long)]
     #[serde(default)]
     pub ignore_resources: bool,
-    // TODO(WAL-911): Change the docstrings when the default behavior changes.
     /// Mark the blob/quilt as deletable.
     ///
     /// Deletable blobs/quilts can be removed from Walrus before their expiration time.
     ///
-    /// *This will become the default behavior in v1.33.*
+    /// This flag is currently optional as new blobs are created as deletable by default since
+    /// v1.33; however, it is still recommended to explicitly use the flag when storing new
+    /// deletable blobs.
     #[arg(long, conflicts_with = "permanent")]
     #[serde(default)]
     pub deletable: bool,
```

Comment:
Maybe we should issue a warning to the console if `--deletable` is used informing the user that we are deprecating it.

Reply:
I'm honestly not sure if we should remove the option. It's always nice to be able to explicitly request the default value to be unaffected by changes to this default. 

We could maybe hide it in a future version though, when we also remove the warning about no specific persistence being set explicitly.